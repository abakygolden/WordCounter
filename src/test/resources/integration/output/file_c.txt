CALIENTE 1
CALL 1
CAN 2
CANNOT 1
CHRISTIAN 3
CIERTAMENTE 1
CIG 2
CLOSER 1
COBRE 1
COLD 1
COMIENZA 1
COMING 1
COMMODO 1
COMO 2
COMPARARLA 1
CON 4
CONDIMENTUM 1
CONFIANZA 1
CONGUE 5
CONOCERLO 1
CONSECTETUR 1
CONSEQUAT 1
CONVALLIS 1
COSAS 1
COULD 2
CRAS 1
CUALQUIER 2
CUANDO 1
CUIDARLA 1
CURABITUR 1