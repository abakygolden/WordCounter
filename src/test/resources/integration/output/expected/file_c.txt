CABBAGE 4
CACTUS 4
CAESAR 16
CAKE 4
CAKES 4
CALZONE 4
CANNOLI 4
CANTALOUPE 4
CAPRESE 4
CEVICHE 4
CHAYOTE 4
CHEESE 8
CHEESECAKE 4
CHEESESTEAK 4
CHERRY 4
CHICKEN 76
CHILI 4
CHIPS 4
CHOCOLATE 8
CHOWDER 4
CLAM 4
CLEMENTINE 4
CLOUDBERRY 4
COBB 4
COCONUT 4
CORNED 4
CRAB 4
CRANBERRY 4
CROISSANT 4
CURRANT 4
CUSTARD 4